### ISSUE TITLE
Is it possible to access relative paths?

### ISSUE THREAD

**Issue by jamesla** [neillturner/kitchen-ansible#233]
I'm just trying to get the ansible_vault_password_file config setting working and was just wondering whether there is anyway to reference a relative path (in my case a home directory)?

I have tried 
`ansible_vault_password_file: ~/.vault_pass.txt
`
 and 
`ansible_vault_password_file: /home/$USER/.vault_pass.txt 
`
but neither are working. Is there a way to do this currently?

**Comment by neillturner** [comment]
i believe those should work. 
run with -l debug to see the debug messages to see what it is copying.
it copy the file to the sandbox minus the leading . and then the sandbox is copies to the server and then ansible run

**Comment by jamesla** [comment]
Hi Neil here is the exception - any ideas?

```
D      ---Nested Exception---
D      Class: Kitchen::ActionFailed
D      Message: Failed to complete #converge action: [No such file or directory @ rb_sysopen - ~/.vault_pass.txt]
D      ----------------------
D      ------Backtrace-------
D      /opt/rbenv/versions/2.4.1/lib/ruby/2.4.0/fileutils.rb:1288:in `initialize'
D      /opt/rbenv/versions/2.4.1/lib/ruby/2.4.0/fileutils.rb:1288:in `open'
D      /opt/rbenv/versions/2.4.1/lib/ruby/2.4.0/fileutils.rb:1288:in `copy_file'
D      /opt/rbenv/versions/2.4.1/lib/ruby/2.4.0/fileutils.rb:429:in `copy_file'
D      /opt/rbenv/versions/2.4.1/lib/ruby/2.4.0/fileutils.rb:356:in `block in cp'
D      /opt/rbenv/versions/2.4.1/lib/ruby/2.4.0/fileutils.rb:1460:in `block in fu_each_src_dest'
D      /opt/rbenv/versions/2.4.1/lib/ruby/2.4.0/fileutils.rb:1476:in `fu_each_src_dest0'
D      /opt/rbenv/versions/2.4.1/lib/ruby/2.4.0/fileutils.rb:1458:in `fu_each_src_dest'
D      /opt/rbenv/versions/2.4.1/lib/ruby/2.4.0/fileutils.rb:355:in `cp'
D      /opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/kitchen-ansible-0.46.4/lib/kitchen/provisioner/ansible_playbook.rb:1130:in `prepare_ansible_vault_password_file'
D      /opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/kitchen-ansible-0.46.4/lib/kitchen/provisioner/ansible_playbook.rb:280:in `create_sandbox'
D      /opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/test-kitchen-1.16.0/lib/kitchen/provisioner/base.rb:65:in `call'
D      /opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/test-kitchen-1.16.0/lib/kitchen/instance.rb:384:in `block in converge_action'
D      /opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/test-kitchen-1.16.0/lib/kitchen/instance.rb:527:in `synchronize_or_call'
D      /opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/test-kitchen-1.16.0/lib/kitchen/instance.rb:489:in `block in action'
D      /opt/rbenv/versions/2.4.1/lib/ruby/2.4.0/benchmark.rb:293:in `measure'
D      /opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/test-kitchen-1.16.0/lib/kitchen/instance.rb:488:in `action'
D      /opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/test-kitchen-1.16.0/lib/kitchen/instance.rb:380:in `converge_action'
D      /opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/test-kitchen-1.16.0/lib/kitchen/instance.rb:359:in `block in transition_to'
D      /opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/test-kitchen-1.16.0/lib/kitchen/instance.rb:358:in `each'
D      /opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/test-kitchen-1.16.0/lib/kitchen/instance.rb:358:in `transition_to'
D      /opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/test-kitchen-1.16.0/lib/kitchen/instance.rb:135:in `converge'
D      /opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/test-kitchen-1.16.0/lib/kitchen/command.rb:197:in `public_send'
D      /opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/test-kitchen-1.16.0/lib/kitchen/command.rb:197:in `run_action_in_thread'
D      /opt/rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/test-kitchen-1.16.0/lib/kitchen/command.rb:169:in `block (2 levels) in run_action'
D      ----End Backtrace-----

```

**Comment by neillturner** [comment]
the code does a 
FileUtils.cp(ansible_vault_password_file, tmp_ansible_vault_password_file_path)
and it can't open the file. 
it could be a permissions problem or something. not a bug in the code. 
i suggest you workaround by doing a 
  cp ~/.vault_pass.txt   /somewhereelse/vault_pass.txt
and access it there.   if that works then you need to look at your permissions or something

**Comment by neillturner** [comment]
this should be resolved by Expand the path to the ansible vault password file #234  fixed in version 0.46.5
